src/encoded/types/imaging.py

```diff
@@ -40,22 +47,41 @@ def display_title(self, request, target=None, labeled_probe=None, other_probes=N
                 target_title = request.embed(a_target, '@@object').get('display_title')
```

Comment:
Can you change from using request.embed  to get_item_or_none here (which I guess was pre-existing) and the following 2 instances - while they are really essentially the same the preferred usage is get_item_or_none - mainly I think due to better error handling?

Reply:
I just used what was there before, didn't know that this was the preferred usage. Changed now.